api/handlers.py

```diff
@@ -0,0 +1,45 @@
+from flask import jsonify
+import MetaTrader5 as meta
+from datetime import datetime
+
+def get_open_orders():
```

Comment:
this should be obtained from IAccount class where you added it. See example of dependency injection pattern where here:

https://github.com/BensGitHub2022/MT5-Expert-Advisor/blob/ben/webservice_example/src/web_service/web_service.py

you already added these methods here:
![image](https://github.com/BensGitHub2022/MT5-Expert-Advisor/assets/115581494/e883c479-5602-4923-ab0e-f6e3efbd9c3e)

Reply:
You should make the methods you created in IAccount get called in flask

Reply:
Side note:

I wasn't able to call these when I switched NODE_ENV=production. I tried doing this by adding NODE_ENV in my system variables and by making a '.env' file like you suggested.